src/main/java/de/taimos/pipeline/aws/AwsSdkResponseToJson.java

```diff
@@ -0,0 +1,24 @@
+package de.taimos.pipeline.aws;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.util.Map;
+
+/**
+ * Jenkins doesn't support returning the API response directly.
+ * Converting it into a map/list construct allows the full API response to be returned to be used in the jenkinsfile.
+ */
+public class AwsSdkResponseToJson {
```

Comment:
Why is this needed in the presign step?

Reply:
its not. I checked it out from a different branch (not master)